recommender_experiments/exp_recommender_master.py

```diff
@@ -3,19 +3,28 @@
 
 # Number of times to run each experiment for each setting
 # e.g., for each setting of recommender, graph,
-#  and top_k_recommendations, select N randomly sampled entities
-#  to compute the recommender hit ratio on that entity.
-N = 136
+# and top_k_recommendations, select N randomly sampled entities
+# to compute the recommender hit ratio on that entity.
+N = 150
 
-recommenders = ["pixie", "popular", "random"]
+recommenders = ["pixie", "random", "popular"]
 
 # The name of the ErdosRenyi graph is of the form
 # ErdosRenyi_[num entities]_[num items]_[num edges]_[graph to draw edge weights from].
 graphs = [
```

Comment:
in all these graphs, the first param, min_threshold isn't used at all? did you forget to include it in some computation, or if it's not needed maybe you can remove it?

Reply:
No it is used. It's unpacked in the for-loop below and passed as a command line argument to `exp_recommender.py`